README.md

```diff
@@ -153,3 +153,23 @@ register - A list of information designed to be an accurate and up-to-date sourc
 territory - An administrative or geographical entity that isn't recognised as a country by the UK.
 
 territory register - A list of British English-language names and descriptive terms for political, administrative and geographical entities that aren’t recognised as countries by the UK.
+
+## Tracking
+
+We use Google Analytics software to collect information about how your users interact with this component. We do this to help make sure the component is meeting the needs of its users and to help us make improvements.
+
+The Google Analytics cookie we set stores information about:
+
+- how many users enter country data using the component the hostname of the service which the component is implemented within
```

Comment:
I think the grammar is off here.

Also: After reading this, I'm still not clear what it does.

Ideally we'd be explicit, as this is information a service would need to add to their cookie policy.

Reply:
@elliecraven can I get your input here, please

Reply:
Is this all it collects? Does it also collect ip address (anonymised?), location, device information?